### DISCUSSION TITLE
运行  ./examples/aishell/asr0/run.sh 报错

### DISCUSSION THREAD

**Discussion by dabaoyasu** [PaddlePaddle/PaddleSpeech#1133] [Q&A] [answered]
请求帮助，在 examples/aishell/asr0/ 下面，运行 bash ./run.sh 之后，报如下错误，会是什么原因？

Aishell data preparation done.
using 4 gpus...
usage: launch.py [-h] [--log_dir LOG_DIR] [--backend BACKEND]
                 [--nproc_per_node NPROC_PER_NODE] [--run_mode RUN_MODE]
                 [--gpus GPUS] [--selected_gpus GPUS] [--ips IPS]
                 [--servers SERVERS] [--workers WORKERS]
                 [--heter_workers HETER_WORKERS]
                 [--heter_devices HETER_DEVICES] [--worker_num WORKER_NUM]
                 [--server_num SERVER_NUM]
                 [--heter_worker_num HETER_WORKER_NUM] [--http_port HTTP_PORT]
                 [--elastic_server ELASTIC_SERVER] [--job_id JOB_ID] [--np NP]
                 [--scale SCALE] [--host HOST] [--force FORCE]
                 training_script ...
launch.py: error: unrecognized arguments: -u
Namespace(ckpt_dir='exp/deepspeech2/checkpoints', dst_model='exp/deepspeech2/checkpoints/avg_1.pdparams', max_epoch=65536, min_epoch=0, num=1, val_best=True)
Traceback (most recent call last):
  File "/users/xunhongsuhuanjie/deepspeech2/PaddleSpeech/utils/avg_model.py", line 116, in <module>
    main(args)
  File "/users/xunhongsuhuanjie/deepspeech2/PaddleSpeech/utils/avg_model.py", line 43, in main
    sort_idx = np.argsort(val_scores[:, 1])
IndexError: too many indices for array: array is 1-dimensional, but 2 were indexed
Failed in avg ckpt!

**Comment by zh794390558** [comment, marked as answer]
我们应该没有用 launch.py 文件，麻烦按照提供的方式跑下。 https://github.com/PaddlePaddle/PaddleSpeech/tree/develop/examples/aishell/asr0

**Reply by dabaoyasu** [reply]
问题解决，非常感谢！
